### PR TITLE
Add email option

### PR DESCRIPTION
**Pull request by willhlaw**
Add email input to the bottom of the modal to email the post. 

New configs that are optional:

- EMAIL_TO=<randomized-wordpress-email@post.wordpress.com>
- GMAIL_USER=<someone@gmail.com>
- GMAIL_PWD=<app-password> # example: xqtxywjleboczism

### CODE REVIEW

File: sendmail.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    msg['Subject'] = subject
+    msg['From'] = gmail_user
+    msg['To'] = recipient
+
+    server = smtplib.SMTP_SSL('smtp.gmail.com', 465)
+    server.ehlo()
+    server.login(gmail_user, gmail_password)
+    server.send_message(msg)
+    server.close()
```
Comment (willhlaw) [inline sendmail.py:20]: Very simple email code...that relies on gmail.

File: app.py [hunk 2]
```diff
@@ -129,24 +171,214 @@ async def command(ack, body, respond, client, logger):
...
+            "element": {
+                "type": "plain_text_input",
+                "action_id": "email-action",
+                "initial_value": config('EMAIL_TO', default='')
+            },
+            "label": {
+                "type": "plain_text",
+                "text": "Email"
+            }
+        })
```
Comment (willhlaw) [inline app.py:380]: Add to break up the building of the slack blocks to conditionally add the email block.

**Review by wolfpackt99 — APPROVED** [review]

File: app.py (hunk 2 above)
Comment (wolfpackt99) [reply app.py:386]: so this emails a copy to the address specified?

**Review by yankeestom — APPROVED** [review]

File: app.py (hunk 2 above)
Comment (willhlaw) [reply app.py:386]: @wolfpackt99 correct. If GMAIL_USER config exists, then the email input block appears in the modal. Then if that input field, email_to has a value, then the post is emailed.

**Pull request merged**